playground/lib/pages/banner.dart

```diff
@@ -21,3 +21,37 @@ class BannerPage extends StatelessWidget {
     ));
   }
 }
+
+@RoutePage()
+class BannerFullPage extends StatelessWidget {
```

Comment:
Just thinking out loud but a future/flexible solution for this Minimal vs. Full is to use query params instead of 2 different pages.

So a query param like `?style=minimal` for this. Then in the future if we want to allow for tons of different displays/properties to be set from the docs we just pass in params?

Reply:
Ok, let me try that in another PR